### PR TITLE
[DE] Add `osu!tourney/Spectator_usage`, `osu!tourney/Troubleshooting` and `osu!tourney`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
This is part of a series that aims to add all articles under `osu!_tournament_client`.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!_tournament_client/osu!tourney/Troubleshooting/de.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Kopiere die Datei `osu!.exe` in einen leeren Ordner und führe sie aus.
+
+## Meine osu!-Fenster sind nicht richtig ausgerichtet!
+
+Deaktiviere jegliche sekundären Bildschirme! osu!tourney läuft nur auf dem primären Monitor. Stelle sicher, dass der primäre Bildschirm eine größere Auflösung als der Client hat (standardmäßig 1280 x 720).
+
+## osu!tourney öffnet sich nicht, sondern gibt eine Fehlermeldung aus und/oder schließt sich!
+
+- Überprüfe, dass du in osu! eingeloggt bist und dass die Checkboxen `Benutzername speichern` und `Passwort speichern` gesetzt sind.
```
Comment (Pasi4K5) [inline wiki/osu!_tournament_client/osu!tourney/Troubleshooting/de.md:13]:
```suggestion
- Stelle sicher, dass du in osu! eingeloggt bist und dass die Checkboxen `Benutzername speichern` und `Passwort speichern` gesetzt sind.
```

File: wiki/osu!_tournament_client/osu!tourney/Troubleshooting/de.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## Die Teamnamen werden nicht angezeigt!
+
+Siehe die vorherige Antwort.
+
+## Wie übertrage ich mein Turnier an Streaming-Dienste wie Twitch?
+
+Wir empfehlen die Verwendung von [Open Broadcaster Software](https://obsproject.com/), aber jedes andere Streaming-Programm, das den Bildschirm aufnimmt, kann ebenso benutzt werden.
+
+Die Auflösung des oberen Teils von osu!tourney kann in der Datei `tournament.cfg` angepasst werden, aber standardmäßig ist sie `1280 x 720`. Überprüfe, dass die Auflösung deines Monitors größer als dieser Wert ist. Siehe die [Einrichtungsanleitung](/wiki/osu!tourney/Setup) für Informationen darüber, wie sich die Auflösung ändern lässt.
```
Comment (Pasi4K5) [inline wiki/osu!_tournament_client/osu!tourney/Troubleshooting/de.md:47]:
```suggestion
Die Auflösung des oberen Teils von osu!tourney kann in der Datei `tournament.cfg` angepasst werden, aber standardmäßig ist sie `1280 x 720`. Stelle sicher, dass die Auflösung deines Monitors größer als dieser Wert ist. Siehe die [Einrichtungsanleitung](/wiki/osu!tourney/Setup) für Informationen darüber, wie sich die Auflösung ändern lässt.
```

File: wiki/osu!_tournament_client/osu!tourney/Troubleshooting/de.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+Vergesse nicht, den Ausschnitt einzustellen, um das schwarze Bedienfeld aus dem Stream zu entfernen!
```
Comment (Pasi4K5) [inline wiki/osu!_tournament_client/osu!tourney/Troubleshooting/de.md:49]:
```suggestion
Vergiss nicht, den Ausschnitt einzustellen, um das schwarze Bedienfeld aus dem Stream zu entfernen!
```

**Review by Pasi4K5 — CHANGES_REQUESTED** [review]

File: wiki/osu!_tournament_client/osu!tourney/Spectator_usage/de.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+Die Funktionalität jedes Buttons wird unten beschrieben:
+
+- `Sync music`: Der Client versucht, die Musik zu den Hitsounds zu resynchronisieren.
+- `Toggle annotation`: Aktiviert/deaktiviert die Anmerkung. Das wird unten näher beschrieben.
+- `Panic`: Drücke das, wenn etwas schief läuft, z. B. ein Fenster beobachtet den Nutzer nicht oder ein Fenster stürzt ab. Dies wird alle osu!-Instanzen reinitialisieren.
+- `Exit`: Fährt den Client herunter.
+
+Die Funktionalität jeder Textbox wird unten beschrieben:
+
+- `Annotation text`: Der Text, um eine Anmerkung anzuzeigen.
```
Comment (FamousPig) [inline wiki/osu!_tournament_client/osu!tourney/Spectator_usage/de.md:25]:
```suggestion
- `Annotation text`: Der Text, der als Anmerkung angezeigt wird.
```

File: wiki/osu!_tournament_client/osu!tourney/Spectator_usage/de.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+- `Best Of`: Aktualisiert die Sternanzahl im oberen Teil mit der Menge an Spielen, die jedes Team braucht, um zu gewinnen.
+
+![Wenn ein Raum korrekt erstellt wurde, wird dies anstatt der Anweisungen aufgelistet](img/Osutourneyroomlist.png)
```
Comment (FamousPig) [inline wiki/osu!_tournament_client/osu!tourney/Spectator_usage/de.md:28]:
```suggestion
![Wenn ein Raum korrekt erstellt wurde, wird dieser anstelle der Anweisungen aufgelistet](img/Osutourneyroomlist.png)
```

File: wiki/osu!_tournament_client/osu!tourney/Troubleshooting/de.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+Ändere den Namen oder lösche die Datei `tournament.cfg`.
+
+## Mein Client ist kein Zuschauer!
+
+Klicke auf den Raumnamen auf dem unteren schwarzen Panel, sodass er **fett gedruckt** erscheint.
+Wenn der Client immer noch nicht zuschaut, klicke auf den Button `Panic`.
+
+## Die Räume tauchen nicht auf!
+
+Es kann passieren, dass dein Raum inkorrekt benannt wurde oder du das falsche Akronym in der Datei `tournament.cfg` benutzt hast.
```
Comment (FamousPig) [inline wiki/osu!_tournament_client/osu!tourney/Troubleshooting/de.md:36]:
```suggestion
Es ist möglich, dass dein Raum inkorrekt benannt wurde oder du das falsche Akronym in der Datei `tournament.cfg` benutzt hast.
```

to me, `kann passieren` signifies something happening outside of the readers control, whereas incorrect naming is very much human error.

File: wiki/osu!_tournament_client/osu!tourney/Troubleshooting/de.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+Die Auflösung des oberen Teils von osu!tourney kann in der Datei `tournament.cfg` angepasst werden, aber standardmäßig ist sie `1280 x 720`. Überprüfe, dass die Auflösung deines Monitors größer als dieser Wert ist. Siehe die [Einrichtungsanleitung](/wiki/osu!tourney/Setup) für Informationen darüber, wie sich die Auflösung ändern lässt.
+
+Vergesse nicht, den Ausschnitt einzustellen, um das schwarze Bedienfeld aus dem Stream zu entfernen!
+
+## Mein Problem/meine Frage ist hier nicht aufgeführt! Was kann ich tun?
+
+Versichere dich, dass osu! nicht als Administrator ausgeführt wird, es sei denn, es fragt dich selbst danach, und dass es auf die neueste Version von CuttingEdge aktualisiert ist.
+
+Sende eine E-Mail an [tournaments@ppy.sh](mailto:tournaments@ppy.sh), sofern du ein Problem hast, das hier nicht in der Liste ist. Stelle sicher, dass die E-Mail beschreibend ist und stelle wenn möglich Screenshots zur Verfügung.
```
Comment (FamousPig) [inline wiki/osu!_tournament_client/osu!tourney/Troubleshooting/de.md:55]:
```suggestion
Sende eine E-Mail an [tournaments@ppy.sh](mailto:tournaments@ppy.sh), sofern du ein Problem hast, das hier nicht aufgelistet ist. Stelle sicher, dass die E-Mail beschreibend ist und stelle wenn möglich Screenshots zur Verfügung.
```

Unrelated to suggestion:

I keep wondering whether we should make a note here that email communication should be happening in english where possible. I assume the people on the other end of these addresses don't necessarily speak german.

**Review by FamousPig — CHANGES_REQUESTED** [review]

**Comment by The-Last-Cookie** [comment]
> I keep wondering whether we should make a note here that email communication should be happening in english where possible. I assume the people on the other end of these addresses don't necessarily speak german.

You are definitely not alone with this idea. For now, I think it should be fine because team members and moderators will answer with an approproate e-mail that they don't speak the respective language. Ideally, they could link to other members of the community which in fact speak that language. Some team wiki pages list people and the languages they speak as far as I know.

Nevertheless, we can have a discussion about this with other contributors. If we decide to use this consequently for all articles, I would like to see a rule being added to the ASC.

**Review by FamousPig — APPROVED** [review]
Just double checked and everything looks good to go from my perspective. If we decide to add the note discussed above, it would be better to do it  in one fell swoop across all articles.

**Review by TPGPL — APPROVED** [review]

**Pull request merged**